css/types/color.json

```diff
@@ -122,7 +122,8 @@
               "spec_url": "https://drafts.csswg.org/css-color-5/#relative-color-function",
               "support": {
                 "chrome": {
-                  "version_added": "119"
+                  "version_added": "119",
+                  "notes": "Supports <code>r g b</code> and <code>x y z</code> origin color channel values for all colorspaces."
```

Comment:
From this note it is unclear that supporting both is actually a bug and not a feature.

I am also unsure if authors need to learn about this because everything that is correct does work and only incorrect input is mistakenly allowed.

imho it would be fine to omit this.

---

chromium issue : https://bugs.chromium.org/p/chromium/issues/detail?id=1517181&q=component%3ABlink%3ECSS%20color&can=2

Reply:
Hrm, yeah, you are probably right on this one; it possibly might make the correct behavior as per spec sound like it is a bug. I'll work on this tomorrow, and probably remove it.